src/cromshell/update_server/command.py

```diff
@@ -0,0 +1,41 @@
+import json
+import logging
+import os
+from pathlib import Path
+
+import click
+
+LOGGER = logging.getLogger(__name__)
+CROMWELL_SERVER_KEY = "cromwell_server"
+
+
+@click.command(name="update-server")
+@click.argument("cromwell_server_url", required=True, nargs=1)
+@click.pass_obj
+def main(config, cromwell_server_url):
+    """Update the default Cromwell server in the config file"""
+
+    # location of the config file
+    cromshell_config_path = os.path.join(
+        config.config_dir, config.CROMSHELL_CONFIG_FILE_NAME
+    )
+    assert os.access(
+        cromshell_config_path, mode=os.W_OK
+    ), f"Cannot write to Cromshell config file {cromshell_config_path}"
+
+    # the contents of the config file as a dict
+    cromshell_config_dict = config.cromshell_config_options
```

Comment:
It isn't [good practice](https://docs.quantifiedcode.com/python-anti-patterns/readability/putting_type_information_in_a_variable_name.html) to use type suffix when naming variables, it would be best to rename `cromshell_config_dict`

Reply:
Haha, okay I tend to to that, but looks like you're right.  What name would you recommend?  Happy to use whatever.  I guess I could do this 
```python
cromshell_config: Dict = config.cromshell_config_options
```
though I think it looks kind of strange

Reply:
I'm going to go with that for now.  I want somebody who reads the code to understand that `config.cromshell_config_options`, which could be anything, is actually a Dict

Reply:
Nice, I didn't know you could specify the data structure of a variable in that way

Reply:
Well... it's a "type hint"... so it's not enforced in any way.  It's just a hint for the reader (and the python IDE), but it can be helpful in a case like this (even though it looks a bit weird)